src/Json_Decode.res

```diff
@@ -68,16 +68,16 @@ let array = (decode, . json) => {
       let value = decode(. %raw("json[i]"))
       target->Array.unsafe_set(i, value)
     } catch {
-    | DecodeError(msg) => raise(DecodeError(j`${msg}\n\tin array at index $i`))
+    | DecodeError(msg) => raise(DecodeError(`${msg}\n\tin array at index $i`))
```

Comment:
Is `$i` now an accepted form of normal interpolated strings, or is this an error that will just print `$i` instead of the value of `i` stringified?

Reply:
`$i` is ignored, as this follows the JS string interpolation spec

Reply:
So this is an error

Reply:
That's why we recommend to check in the build artifacts

Reply:
Fixed now.